standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/utils/SecurityUtils.java

```diff
@@ -74,7 +73,7 @@
 public class SecurityUtils {
   private static final Logger LOG = LoggerFactory.getLogger(SecurityUtils.class);
 
-  public static UserGroupInformation getUGI() throws LoginException, IOException {
```

Comment:
The `LoginException` would never be thrown, I consider to remove all uses of it in a new PR because it's not quite relative to current PR.